rest-server/src/templates/yarnContainerScript.mustache

```diff
@@ -28,10 +28,13 @@ BASH_XTRACEFD=13
 
 function exit_handler()
 {
-  printf "%s %s\n" \
-    "[DEBUG]" "Yarn container exit handler: EXIT signal received in yarn container, exiting ..."
-  pid=$(docker inspect --format={{{ inspectFormat }}} $docker_name) && kill -9 $pid ||\
-    printf "%s %s\n"  "[DEBUG]" "Yarn container exit handler tries to kill the container $docker_name that does not exist. It probably has already exited."
+  pid=$(docker inspect --format={{{ inspectFormat }}} $docker_name 2>/dev/null)
+  if [ $pid ]; then
+    printf "%s %s\n" \
+      "[DEBUG]" "Yarn container exit handler: EXIT signal received in yarn container, exiting ..."
+    kill -9 $pid ||\
+      printf "%s %s\n"  "[DEBUG]" "Yarn container exit handler tries to kill the container $docker_name that does not exist. It probably has already exited."
```

Comment:
>that does not exist. It probably has already exited. [](start = 104, length = 52)

but failed. Maybe it has already exited.

Reply:
log changed. done